packages/fast-components/src/text-field/text-field.template.ts

```diff
@@ -0,0 +1,57 @@
+import { html, when } from "@microsoft/fast-element";
+import { ref } from "@microsoft/fast-element/dist/directives";
```

Comment:
I think this is now exported from the root so you should be able to leave off the `dist/directives`.

Reply:
Yep, this one is now available directly from `@microsoft/fast-element` as of the PR I merged this morning.

Reply:
I'll update now that it's merged.